src/DependencyInjection/MongoDbBundleExtension.php

```diff
@@ -93,7 +101,24 @@ private function defineConnections(array $connections): void
         $this->containerBuilder->setAlias('mongo.connection', new Alias('mongo.connection.' . array_keys($connections)[0], true));
     }
 
-    private function attachDataCollectionListenerToEventManager(): void
+    /**
+     * Normalizes the driver options array
+     *
+     * @param array $clientsConfig
+     *
+     * @return array|null
+     */
+    private function normalizeDriverOptions(array $clientsConfig)
+    {
+        $driverOptions            = $clientsConfig['driver_options'] ?? [];
+        $driverOptions['typeMap'] = ['root' => 'array', 'document' => 'array'];
+        if (isset($driverOptions['context'])) {
+            $driverOptions['context'] = new Reference($driverOptions['context']);
+        }
+        return $driverOptions;
+    }
+
+    private function attachDataCollectionListenerToEventManager()
```

Comment:
You lost the return type here.

Reply:
you are right, it should be:

* @return array|mixed